### PR TITLE
Changed documentation of Api v3

### LINKED ISSUE #6559: v3 api version-update method not working
## Details
I am trying to activate/deactivate versions via the v3 api
https://docs.readthedocs.io/en/stable/api/v3.html#version-update

This is implemented per your docs, however does not appear to be
it working yet as this always returns a 404.

```
$ lftools rtd project-version-update o-ran-sc-doc master false
{'active': 'false', 'privacy_level': 'public'}
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://readthedocs.org/api/v3/projects/o-ran-sc-doc/version/master/
```

Code on our end:

```
    def project_version_update(self, project, version, active,
                               privacy_level):
        """Edit a version.

        :param project: The project slug
        :param version: The version slug
        :param active: 'true' or 'false'
        :param privacy_level: 'public' or 'private'
        :return: {result}
        """
        data = {
            'active': active,
            'privacy_level': privacy_level
        }
        print(data)

        json_data = json.dumps(data)
        result = self.patch('projects/{}/version/{}/'.format(project, version),
                            data=json_data)
        return result
```

## Expected Result

I can activate versions via the API 

## Actual Result

API method returns 404.

### PR DESCRIPTION
**Pull request by Blackcipher101**
Fixes #6559

### CODE REVIEW

**Comment by Blackcipher101** [comment]
Where are the changes needed

**Comment by saadmk11** [comment]
You need to update the API endpoint and also remove the trailing comma from the json from here:
https://github.com/readthedocs/readthedocs.org/blob/master/docs/api/v3.rst#version-update

**Comment by Blackcipher101** [comment]
@saadmk11 Should this 
``https://readthedocs.org/api/v3/projects/pip/versions/`` to ``https://readthedocs.org/api/v3/pprojects/{project}/versions/{version}``

**Comment by Blackcipher101** [comment]
@stewed I will do the needful

File: docs/api/v3.rst [hunk 1]
```diff
@@ -161,7 +161,7 @@ Project details
             },
             "_links": {
                 "_self": "/api/v3/projects/pip/",
-                "versions": "/api/v3/projects/pip/versions/",
+                "versions": "/api/v3/project/pip/version/",
```
Comment (stsewd) [inline docs/api/v3.rst]: you shouldn't change this

File: docs/api/v3.rst [hunk 2]
```diff
@@ -294,7 +294,7 @@ Versions listing
 Version detail
 ++++++++++++++
 
-.. http:get:: /api/v3/projects/(string:project_slug)/versions/(string:version_slug)/
+.. http:get:: /api/v3/projects/{project}/versions/{version}
```
Comment (stsewd) [inline docs/api/v3.rst]: neither this

File: docs/api/v3.rst [hunk 3]
```diff
@@ -346,7 +346,7 @@ Version detail
 Version update
 ++++++++++++++
 
-.. http:patch:: /api/v3/projects/(string:project_slug)/version/(string:version_slug)/
+.. http:patch:: /api/v3/projects/{project}/versions/{version}
```
Comment (stsewd) [inline docs/api/v3.rst]: you shouldn't remove the types

File: docs/api/v3.rst [hunk 4]
```diff
@@ -315,7 +315,7 @@ Version detail
             "identifier": "3a6b3995c141c0888af6591a59240ba5db7d9914",
             "ref": "19.0.2",
             "built": true,
-            "active": true,
+            "active": true
```
Comment (stsewd) [inline docs/api/v3.rst]: this shouldn't change

File: docs/api/v3.rst (hunk 2 above)
Comment (Blackcipher101) [reply docs/api/v3.rst]: Done

File: docs/api/v3.rst (hunk 1 above)
Comment (Blackcipher101) [reply docs/api/v3.rst]: Done

File: docs/api/v3.rst (hunk 4 above)
Comment (Blackcipher101) [reply docs/api/v3.rst]: Done

File: docs/api/v3.rst (hunk 3 above)
Comment (Blackcipher101) [reply docs/api/v3.rst]: should it go like ``(string:project_slug)/versions/(string:version_slug)/`` or ``(string:{projects})/versions/(string:{version})/

Comment (stsewd) [reply docs/api/v3.rst]: Like the first one

File: docs/api/v3.rst [hunk 5]
```diff
@@ -346,7 +346,7 @@ Version detail
 Version update
 ++++++++++++++
 
-.. http:patch:: /api/v3/projects/(string:project_slug)/version/(string:version_slug)/
+.. http:patch:: /api/v3/projects/(string:{projects})/versions/(string:{version})/
```
Comment (stsewd) [inline docs/api/v3.rst]:
```suggestion
.. http:patch:: /api/v3/projects/(string:project_slug)/versions/(string:version_slug)/
```

**Pull request merged**